examples~/quickstart/client/client.csproj

```diff
@@ -1,16 +1,16 @@
-<Project Sdk="Microsoft.NET.Sdk">

```

Comment:
are the C# SDK quickstarts still important if we don't have a separate C# SDK anymore?

Reply:
We still do though - this PR doesn't remove it, as it keeps both NuGet and Unity packages working.

Reply:
Ah okay got it